src/debugger/debugger.c

```diff
@@ -63,6 +63,8 @@ struct mDebuggerModule* mDebuggerCreateModule(enum mDebuggerType type, struct mC
 		free(debugger);
 		return 0;
 		break;
+	case DEBUGGER_ACCESS_LOGGER:
```

Comment:
This is definitely not correct. See above about the "more work" comment. I need to decide what the right behavior is, but it's not this.

Reply:
reverted